var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -27,12 +27,15 @@ class Mesa(MesonPackage):
 
     depends_on('meson@0.45:', type='build')
     depends_on('binutils', type='build')
+    depends_on('bison', type='build')
+    depends_on('flex', type='build')
```

Comment:
I see the bison and flex dependency by where is the gettext dependency coming in?

Reply:
The build was looking for/tried to call `xgettext`. On second thought, this probably should also be a build dependency then.

Reply:
Agreed